addons/website_slides/static/src/less/website_slides.less

```diff
@@ -86,6 +86,10 @@ img.oe_slides_promote_image {
     color: @link-color;
 }
 
+textarea[name=comment] {
```

Comment:
You are going to change the style of every textarea having a name=comment !
Please prefix it, or add a css class !